README.md

```diff
@@ -22,6 +22,7 @@ Each link in this category is scoped to posts tagged with Archivematica.
 - 2019-09-25: Alex Garnett: "Keeping Archivematica Video Filesizes Manageable" [blog post](https://axfelix.github.io/ffv1).
 - 2019-09-16: Goodchild, M. and Hurley, G. at iPres 2019: "Integrating Dataverse and Archivematica for Research Data Preservation." [article](https://ipres2019.org/static/pdf/iPres2019_paper_147.pdf).
 - 2019-08-07: Rice, D., & Schweikert, A. (2019). Microservices in Audiovisual Archives. International Association of Sound and Audiovisual Archives (IASA) Journal, (50), 53–75. https://doi.org/10.35320/ij.v0i50.70.
+- 2019-08-01: Canadian Research Knowledge Network: "CRKN's use of Archivematica" [github repository](https://github.com/crkn-rcdr/Digital-Preservation/tree/master/Archivematica).
```

Comment:
Oh cool! What do you reckon to having a permalink for the page as it is today? e.g. [109063](https://github.com/crkn-rcdr/Digital-Preservation/tree/1090634d377da01ee920849eb32d472425bb3cf7/Archivematica)

Reply:
Oh sure thing, I'll update it, thanks!